src/plugins/index_pattern_field_editor/public/components/flyout_panels/flyout_panels.scss

```diff
@@ -0,0 +1,33 @@
+.fieldEditor__flyoutPanels {
+  height: 100%;
+}
+
+.fieldEditor__flyoutPanel {
+  height: 100%;
+  overflow-y: auto;
+  padding: $euiSizeL;
+
+  &--lightestShade {
+    background-color: $euiColorLightestShade;
+  }
+  &--lightShade {
+    background-color: $euiColorLightShade;
+  }
+  &--mediumShade {
+    background-color: $euiColorMediumShade;
+  }
```

Comment:
We cannot guarantee that there will be proper contrast for any component against any of these background colors. Yes, technically the base text color will be ok, but not if they use any other text color or buttons or charts or logos. Even links will fail. 
<img width="825" alt="Screen Shot 2021-05-27 at 14 17 16 PM" src="https://user-images.githubusercontent.com/549577/119876826-4b549a00-bef6-11eb-99bc-22734dc40b14.png">

Are these shades absolutely necessary?

Reply:
I will restrict to only `$euiColorEmptyShade` and `$euiPageBackgroundColor` as you suggest below. My point of view was that if the consumer decides to use a dark background it is his responsibility to assure the proper contrast of the text against that background.